.github/workflows/tests.yml

```diff
@@ -23,12 +23,15 @@ jobs:
       # Upstream bug causes buildkit pulls to fail so prefetch base images
       # https://github.com/moby/moby/issues/41864
       run:  |
-        echo "_APP_FUNCTIONS_RUNTIMES=php-8.0" >> .env
+        export COMPOSE_INTERACTIVE_NO_CLI
+        export DOCKER_BUILDKIT=1
+        export COMPOSE_DOCKER_CLI_BUILD=1
+        export BUILDKIT_PROGRESS=plain
         docker pull composer:2.0
         docker pull php:8.0-cli-alpine
```

Comment:
Unrelated to your change, but we may not be using `php:8.0-cli-alpine` anymore 🧐 . Do we still need it in the GitHub Action? We can change this in a future PR.